dlpy/transformers/bert_model.py

```diff
@@ -33,8 +33,18 @@
     from transformers import BertTokenizer, BertModel, BertConfig
     from transformers import RobertaTokenizer, RobertaModel, RobertaConfig
     from transformers import DistilBertTokenizer, DistilBertModel, DistilBertConfig
+    
+    from distutils.version import StrictVersion
+    import pkg_resources
+    import warnings
+    if StrictVersion(pkg_resources.get_distribution("transformers").version) > '2.3.0':
+        warn_message = ("You are using a version of the transformers package ("
+                        + pkg_resources.get_distribution("transformers").version + 
+                        ") that has not been tested for compatibility.  Unexpected behavior "
+                        "may occur.")
+        warnings.warn(warn_message,UserWarning)
 except:
-    raise DLPyError("Unable to import from transformers."
+    raise DLPyError("Unable to import from transformers.  "
```

Comment:
can we also add wording indicating that versions missmatches are also another reason failure

Reply:
I am a little confused by this comment.  The only reason we hit this message is if we cannot import the transformers package.  I give a warning message if there are version issues.  I must be misunderstanding what you are asking, so would you mind clarifying?

Reply:
so what happens when you have a not supported version? does it stop execution?

Reply:
It does not stop execution, it just gives a warning message.  The warning message is something like this:  

"You are using a version of the transformers package (X.X.X) that has not been tested for compatibility.  Unexpected behavior may occur."

This is my way of saying, "we haven't tested it, you are on your own".  Not all updates to the transformers package will break our code, so I want to give the user the flexibility to use the version of his/her choice.  At the same time, I don't want to be on the hook for debugging version issues.  This was my compromise.

Reply:
how about adding something like "please install a supported transformers version (i.e., < 2.3.0)."

Reply:
I can do that.  I will update my pull request shortly.

Reply:
thanks.

Reply:
I made the change you requested.  Unfortunately it took several attempts so there appear to be multiple commits.